lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -35,6 +35,25 @@ aaa_authentication_login:
     config_set: "%s aaa authentication login mschapv2 enable"
     default_value: false
 
+aaa_authorization_service:
+  groups:
+    config_get: "show aaa authorization all"
+    config_get_token: ["TABLE_cmd_methods", "ROW_cmd_methods"]
+    # this set is only used when there are groups to configure
+    config_set: "%s aaa authorization %s %s group %s %s"
```

Comment:
It'd be nice to use named parameters (https://github.com/cisco/cisco-network-node-utils/blob/develop/docs/README-develop-best-practices.md#yaml7) but that's not critical for this commit.